app/controllers/moderate_stories_controller.rb

```diff
@@ -0,0 +1,43 @@
+class ModerateStoriesController < ApplicationController
+  before_action :require_logged_in_moderator
+  before_action :find_user_story!
+  before_action :show_title_h1
+
+  def edit
+    @title = "Edit Story"
+
+    if @story.merged_into_story
```

Comment:
This code about story merging can be removed from the `StoriesController#edit` action, only mods are allowed to edit this field.